lib/index.js

```diff
@@ -196,18 +196,7 @@ Queue.prototype._processHead = function() {
   var now = this._schedule.now();
   var toRun = [];
 
-  while (queue.length && queue[0].time <= now) {
-    var el = queue.shift();
-
-    var id = uuid();
-
-    // Save this to the in progress map
-    inProgress[id] = {
-      item: el.item,
-      attemptNumber: el.attemptNumber,
-      time: this._schedule.now()
-    };
-
+  function enqueue(el, id) {
```

Comment:
Actually I don't know if this makes sense to pull into another sub function - it's not really used from anywhere else.

Reply:
It's because I'm rebased on #10 (you can find the reason for the function in the PR). I think I squashed the commit somehow, I'm going to push separate commits

Reply:
👍 We'll probably release them one at a time anyway.